sycl/include/CL/sycl/detail/cl.h

```diff
@@ -0,0 +1,18 @@
+//==---------- common.hpp ----- Common declarations ------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+// Suppress a compiler warning about undefined CL_TARGET_OPENCL_VERSION
```

Comment:
```suggestion
// Suppress a compiler message about undefined CL_TARGET_OPENCL_VERSION
```
I wasn't precise with the original wording - sorry. Could you fix my mistake, please?